src/components/Permalink/Permalink.js

```diff
@@ -36,6 +36,10 @@ const LINE_FILTER = 'PublishedLineName';
 const ROUTE_FILTER = 'TripNumber';
 const OPERATOR_FILTER = 'Operator';
 
+const getUrlParam = param => {
```

Comment:
```suggestion
const getUrlParam = (param = '') => {
```

Reply:
if the param is not undefined, it will be set as empty in the url. That is not what we need.